markdown/3.x/en/documentation/server-side-rendering.md

```diff
@@ -0,0 +1,65 @@
+## Server Side Rendering
+
+Server Side Rendering (SSR) is currently a trending technique which enables pre-rendering a whole page's HTML on the server side before sending it back to the client.
+There are a couple of benefits from using this approach:
+* HTML content is available to web crawlers for scanning which is important for SEO
+* Unified performance, conditioned by the application server, not the user's web browser
+
+With SSR, an application server responds with a page that is viewable but not yet functional.
+Then the client side part of the application renders the HTML once again making it fully functional for the user.
+Opposite to SSR stands CSR (client side rendering) which is the standard way of running web applications.
+With this approach the server returns raw HTML, often containing framework-specific tags, which should not be visible to the user.
+Then the client side application renders the fully viewable page.
+However, the rendered result is not available to scanning bots as they only scan the HTML returned by the server.
+This might be an issue where an application renders business critical content (e.g. an e-shop rendering products) and  has the effect that search engines will never index such applications properly as they have no chance to detect what the real and full page content is.
```

Comment:
```suggestion
This might be an issue where an application renders business critical content (e.g. an e-shop rendering products) and has the effect that search engines will never index such applications properly as they have no chance to detect what the real and full page content is.
```